conf/metadata.php

```diff
@@ -5,3 +5,4 @@
 
 $meta['key'] = array('string');
 $meta['secret'] = array('password');
+$meta['serverId'] = array('string');
```

Comment:
I think server IDs are numeric in case of Discord, will check later if DokuWiki has an option.